backend/migration/qdrant_migration.py

```diff
@@ -217,7 +223,7 @@ def main():
         destination_qdrant_url=args.destination_qdrant_url,
         destination_prefix=args.destination_prefix,
         batch_size=args.batch_size,
-        overwrite=args.overwrite,
+        overwrite=args.overwrite == "True",
```

Comment:
Will this be fine?
```
parser.add_argument(
        "--overwrite",
        type=str,
        help="Overwrite destination collection if exists in separate qdrant",
        required=False,
        default="True",
    )
```

Reply:
Right!
```
parser.add_argument(
        "--overwrite",
        type=bool,
        help="Overwrite destination collection if exists in separate qdrant",
        required=False,
        action="store_true",
    )

    args = parser.parse_args()

    migrate_collection(
        # source args
        source_backend_url=args.source_backend_url,
        source_collection_name=args.source_collection_name,
        source_qdrant_url=args.source_qdrant_url,
        source_prefix=args.source_prefix,
        # destination args
        destination_backend_url=args.destination_backend_url,
        destination_collection_name=args.destination_collection_name,
        destination_qdrant_url=args.destination_qdrant_url,
        destination_prefix=args.destination_prefix,
        batch_size=args.batch_size,
        overwrite=args.overwrite,
    )
```

Via this, if I specify `--overwrite` then it will result as `True` else `False`, right?

Reply:
correct